### PR TITLE
(LVIIICNET-143) As Member I want to choose picture for a project

### PR DESCRIPTION
**Pull request by BohdanZahirnuy**

### CODE REVIEW

File: TheraLang.BLL/Services/ProjectService.cs [hunk 1]
```diff
@@ -10,16 +10,20 @@
...
     {
         private readonly IUnitOfWork _unitOfWork;
+        private readonly IFileService _fileService;
 
-        public ProjectService(IUnitOfWork unitOfWork)
+        public ProjectService(IUnitOfWork unitOfWork, IFileService fileService)
         {
             _unitOfWork = unitOfWork;
+            _fileService = fileService;
+
```
Comment (artur-britvin) [inline TheraLang.BLL/Services/ProjectService.cs]: Empty line)

File: TheraLang.BLL/Services/ProjectService.cs [hunk 2]
```diff
@@ -201,5 +214,6 @@ public async Task<ProjectDto> GetByIdAsync(int id)
                 throw new Exception($"Error when getting project by {nameof(id)} = {id} ", ex);
             }
         }
+
```
Comment (artur-britvin) [inline TheraLang.BLL/Services/ProjectService.cs]: And here)

File: TheraLang.Web/ViewModels/ProjectViewModel.cs [hunk 3]
```diff
@@ -19,5 +20,8 @@ public class ProjectViewModel
         public DateTime ProjectEnd { get; set; }
 
         public decimal DonationTargetSum { get; set; }
+        public string ImgUrl { get; set; }
+        public IFormFile ImgFile { get; set; }
```
Comment (oleksiistetsiouk) [inline TheraLang.Web/ViewModels/ProjectViewModel.cs]: put empty lines between properties or not

File: TheraLang.Web/ViewModels/ProjectDonationViewModel.cs [hunk 4]
```diff
@@ -23,6 +23,9 @@ public class ProjectDonationViewModel
         public decimal DonationTargetSum { get; set; }
 
         public decimal SumLeftToCollect { get; set; }
+
+        public string ImgUrl { get; set; }
+
```
Comment (oleksiistetsiouk) [inline TheraLang.Web/ViewModels/ProjectDonationViewModel.cs]: remove empty line

File: TheraLang.Web/Controllers/ProjectController.cs [hunk 5]
```diff
@@ -70,7 +70,8 @@ public async Task<IEnumerable<ProjectDonationViewModel>> GetAllProjects()
                     Description = p.Description,
                     Details = p.Details,
                     ProjectStart = p.ProjectStart,
-                    ProjectEnd = p.ProjectEnd
+                    ProjectEnd = p.ProjectEnd,
+                    ImgUrl=p.ImgUrl
```
Comment (oleksiistetsiouk) [inline TheraLang.Web/Controllers/ProjectController.cs]:
```suggestion
                    ImgUrl = p.ImgUrl                             
```

File: TheraLang.Web/Controllers/ProjectController.cs [hunk 6]
```diff
@@ -32,7 +32,7 @@ public ProjectController(IProjectService projectService, IUserManagementService
         /// <returns>status code</returns>
         [HttpPost("create")]
         [Authorize]
-        public async Task<IActionResult> CreateProject([FromBody] ProjectViewModel projectModel)
+        public async Task<IActionResult> CreateProject([FromForm] ProjectViewModel projectModel)
         {
```
Comment (oleksiistetsiouk) [inline TheraLang.Web/Controllers/ProjectController.cs]: empty line

File: TheraLang.Web/ClientApp/src/assets/i18n/ua.json [hunk 7]
```diff
@@ -71,7 +71,9 @@
       "owner": "Царь и Властелин Проекта",
       "join": "Приєднатися",
       "days-left": "Днів залишилось",
-      "rised-of": "зібрано з"
+      "rised-of": "зібрано з",
+      "add-image": "Додати зображення"
+
```
Comment (oleksiistetsiouk) [inline TheraLang.Web/ClientApp/src/assets/i18n/ua.json]: empty line

File: TheraLang.DAL/Entities/Project.cs [hunk 8]
```diff
@@ -25,6 +25,7 @@ public class Project
         public int TypeId { get; set; }
 
         public decimal DonationTarget { get; set; }
```
Comment (oleksiistetsiouk) [inline TheraLang.DAL/Entities/Project.cs]:
```suggestion
        public decimal DonationTarget { get; set; }
        
```

**Pull request merged**